src/cr/cube/cube.py

```diff
@@ -603,11 +611,26 @@ def raw_cube_array(self):
         response. Specifically, it includes values for missing elements, any
         MR_CAT dimensions, and any prunable rows and columns.
         """
-        array = np.array(self._flat_values).reshape(self._all_dimensions.shape)
+        array = np.array(self._flat_values).reshape(self.shape)
```

Comment:
What do you think if we change the name `array` in something more specific? I know it's not related to you changes but considering that you touch this line, maybe this can be refactored using a meaningful name.

Reply:
Done, I've used the `raw_cube_array` (same as the name of the property) instead. I'm open to better ideas.